sparkplug_b/stand_alone_examples/c/udt_example/example.c

```diff
@@ -344,7 +344,6 @@ void my_log_callback(struct mosquitto *mosq, void *userdata, int level, const ch
 void publish_births(struct mosquitto *mosq) {
 	// Initialize the sequence number for Sparkplug MQTT messages
 	// This must be zero on every NBIRTH publish
-	seq = 0;
```

Comment:
Also needs a call to `reset_sparkplug_sequence` put in to replace this, preferably around line 363 just before the call to `get_next_payload` for the NBIRTH.